classes/fields/wysiwyg.php

```diff
@@ -71,6 +71,13 @@ public function options() {
 					),
 				),
 			),
+			static::$type . '_editor_height'     => array(
+				'label'           => __( 'Editor Height', 'pods' ),
+				'default'         => '',
+				'type'            => 'number',
+				'depends-on'      => array( static::$type . '_editor' => 'tinymce' ),
+				'number_decimals' => 0,
```

Comment:
We should add further explanation like what this number represents (how many rows? how many px tall?)